packages/address/src/loader.ts

```diff
@@ -10,8 +10,11 @@ import {
 
 import query from './graphqlQuery';
 
-export const loadCountries: (locale: string) => Promise<Country[]> =
-  memoizeAsync(async (locale: string) => {
+export const loadCountries: (
+  locale: string,
+  includeHiddenZones: boolean,
```

Comment:
`loadCountries` is part of the public API of this package as it (and every other export in loader.ts) is exported by the index.ts file. Adding a new required argument to a public API function is a breaking change that would require a new major version.

Please make `includeHiddenZones` an optional parameter, with the default value of `false`. That way we maintain backwards compatibility (i.e. people doing `loadCountries('en')` won't get an error) and this is a minor, non-breaking change.

Reply:
Thank you for calling me out on this!  (My grasp of TypeScript isn't what it should be.)

Have I managed to get it right now, like this?
```patch
diff --git a/packages/address/src/loader.ts b/packages/address/src/loader.ts
index 8e9399945..816fbf63f 100644
--- a/packages/address/src/loader.ts
+++ b/packages/address/src/loader.ts
@@ -12,7 +12,7 @@ import query from './graphqlQuery';

 export const loadCountries: (
   locale: string,
-  includeHiddenZones: boolean,
+  includeHiddenZones?: boolean,
 ) => Promise<Country[]> = memoizeAsync(
   async (locale: string, includeHiddenZones: boolean) => {
     const response = await fetch(GRAPHQL_ENDPOINT, {
@@ -42,8 +42,13 @@ export const loadCountries: (
 export const loadCountry: (
   locale: string,
   countryCode: string,
+  includeHiddenZones?: boolean,
 ) => Promise<Country> = memoizeAsync(
-  async (locale: string, countryCode: string): Promise<Country> => {
+  async (
+    locale: string,
+    countryCode: string,
+    includeHiddenZones: boolean,
+  ): Promise<Country> => {
     const response = await fetch(GRAPHQL_ENDPOINT, {
       method: 'POST',
       headers: HEADERS,
@@ -53,6 +58,7 @@ export const loadCountry: (
         variables: {
           countryCode,
           locale: locale.replace(/-/, '_').toUpperCase(),
+          includeHiddenZones,
         },
       }),
     });
```